mocks_sig.json

```diff
@@ -1,35 +1,35 @@
-  "github.com/Scalingo/go-scalingo.CollaboratorsService": "b2 c5 1f df 12 d0 a9 44 da f5 ee 8c 83 ba 80 3e 32 98 eb 84",
-  "github.com/Scalingo/go-scalingo.ContainerSizesService": "e2 db 25 a1 01 52 dc e9 6f 7b f4 59 68 95 86 b9 5e 39 8a 61",
-  "github.com/Scalingo/go-scalingo.ContainersService": "ef 14 ae 81 c9 b4 6a 13 48 e8 bf 44 7d b5 b2 a6 4d e5 02 11",
-  "github.com/Scalingo/go-scalingo.CronTasksService": "9b d9 a3 ca 9f 9a f7 73 cb b1 aa 80 df 90 21 ff 14 92 b4 4d",
-  "github.com/Scalingo/go-scalingo.DatabasesService": "bf 4d 92 13 a5 11 a5 f6 ad 14 f3 72 81 19 5a 48 65 1e b1 90",
-  "github.com/Scalingo/go-scalingo.DeploymentsService": "aa b5 25 87 ff f0 d2 9f 3a 32 42 2d d6 02 33 5f 63 14 2b 9a",
-  "github.com/Scalingo/go-scalingo.DomainsService": "2f 49 a8 58 f3 ae 8c 4d 4d 6b 1f 47 fc dc db b3 d0 84 ba de",
-  "github.com/Scalingo/go-scalingo.EventsService": "29 cf 29 33 fb 33 ce 86 3b 49 db ae 25 2d 5d 22 d6 60 21 95",
-  "github.com/Scalingo/go-scalingo.KeysService": "b9 5f 5d 8e 88 9d db 82 d6 69 e5 bd 94 6c 7e 08 14 69 23 48",
-  "github.com/Scalingo/go-scalingo.LogDrainsService": "d4 05 d0 5e a7 7f b7 3e 58 19 07 8d 25 c5 af ca dc d3 6e d9",
-  "github.com/Scalingo/go-scalingo.LogsArchivesService": "35 b6 c3 7b 41 1a 47 5c 3c df ab a6 d6 93 1e 6e b1 b1 f9 11",
-  "github.com/Scalingo/go-scalingo.LogsService": "19 3c 84 31 5a a6 7b 8f af 8a 9a 89 d5 5f 9b b8 50 75 9e b7",
-  "github.com/Scalingo/go-scalingo.NotificationPlatformsService": "9b a8 93 13 ed 36 09 92 be dc cb 95 17 02 93 28 18 53 ae 01",
-  "github.com/Scalingo/go-scalingo.NotifiersService": "b7 82 f2 c1 a1 81 70 05 ed 5a 65 c0 56 13 de 6e 2c 13 ca 1f",
-  "github.com/Scalingo/go-scalingo.OperationsService": "28 1a bc 62 4c bd 9f e2 ef 45 90 7f 0d 82 5f 30 23 07 93 2a",
-  "github.com/Scalingo/go-scalingo.RegionMigrationsService": "79 37 8f 49 6a 2f 79 d7 91 a7 6b 5c 8a 8f 66 54 5d 32 0f 5d",
-  "github.com/Scalingo/go-scalingo.RegionsService": "bc d0 b0 22 5b 7c df 31 02 b7 b3 4d 1f b4 9a 88 a2 7c 8a 6a",
-  "github.com/Scalingo/go-scalingo.RunsService": "b3 27 40 57 21 c2 64 f8 08 89 c9 0e 06 45 0d 1b ff 5b 5a 45",
-  "github.com/Scalingo/go-scalingo.SCMRepoLinkService": "bd d9 5a a3 19 18 70 9e 5c 0e 18 a3 76 ea 6e 9d d1 19 18 41",
-  "github.com/Scalingo/go-scalingo.SignUpService": "ae 46 80 88 8e b6 bd c3 ea d9 2c e5 d4 b5 37 ac a9 e8 4f cf",
-  "github.com/Scalingo/go-scalingo.SourcesService": "05 1f 06 73 0a 43 2a 6a 35 d2 b9 41 d2 be 8e 7c ad b0 63 3e",
-  "github.com/Scalingo/go-scalingo.TokensService": "32 2d 05 42 b4 9c a9 3b 1b 6c ce 0e da 18 cf 6f 8f cb fc 2a",
-  "github.com/Scalingo/go-scalingo.UsersService": "9e c5 75 1e aa 05 dd a7 a3 17 5f 4e 82 80 e2 34 55 a2 70 df",
-  "github.com/Scalingo/go-scalingo.VariablesService": "77 27 0b 8a 6b ac c9 c8 c3 b0 e0 f8 cf 7d 6e a0 63 9d 95 03",
+  "github.com/Scalingo/go-scalingo.API": "NOFILE",
```

Comment:
Please call `gomock_generator` a second time. This is due to a known bug in `gomock_generator`.

Reply:
Sorry, I didn't see that